django/db/models/functions.py

```diff
@@ -17,17 +17,18 @@ def __init__(self, *expressions, **extra):
             raise ValueError('Coalesce must take at least two expressions')
         super(Coalesce, self).__init__(*expressions, **extra)
 
-    def as_oracle(self, compiler, connection):
+    def as_oracle(self, compiler, connection, **kwargs):
         # we can't mix TextField (NCLOB) and CharField (NVARCHAR), so convert
         # all fields to NCLOB when we expect NCLOB
+        c = self.copy()
```

Comment:
It's mostly consistent with other copies in the same file, but `clone = ` and `copy = ` is also used. I think it's probably better to use `copy`.